data/mods/BombasticPerks/perks.json

```
@@ -1179,6 +1179,23 @@
     "activated_is_setup": false,
     "activated_eocs": [ "EOC_forge_blood" ]
   },
+  {
+    "type": "mutation",
+    "id": "perk_dexterous_speed",
+    "name": { "str": "Dexterous Speed" },
+    "points": 0,
+    "purifiable": false,
+    "valid": false,
+    "description": "You've found that you can move just a little bit quicker the more dexterous you are.  Unfortunately, this applies comes at a loss to your max carry weight.  Gain 2 speed for every point of DEX away from 8 and lose 4 kg of carry weight.",
+    "category": [ "perk" ],
+    "enchantments": [
+      { "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": { "math": [ "( u_val('dexterity') - 8 ) * 2" ] } } ] },
+      {
+        "condition": "ALWAYS",
+        "values": [ { "value": "CARRY_WEIGHT", "subtract": { "math": [ "( u_val('dexterity') - 8 ) * 4000" ] } } ]
```

```suggestion
        "values": [ { "value": "CARRY_WEIGHT", "add": { "math": [ "( 8 - u_val('dexterity') ) * 4000" ] } } ]
```
Is it intentional that these will provide the opposite effects at less than 8 dex? Otherwise they probably want to be:
```json
{ "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": { "math": [ "max( 0, u_val('dexterity') - 8 ) * 2" ] } } ] },
```

```json
"values": [ { "value": "CARRY_WEIGHT", "add": { "math": [ "min( 0, 8 - u_val('dexterity') ) * 4000" ] } } ]
```